source/extensions/filters/http/ext_proc/processor_state.h

```diff
@@ -49,14 +64,24 @@ class ProcessorState : public Logger::Loggable<Logger::Id::filter> {
   mutableBody(envoy::service::ext_proc::v3alpha::ProcessingRequest& request) const PURE;
 
 protected:
+  Filter* const parent_;
   Http::StreamFilterCallbacks* filter_callbacks_;
   CallbackState callback_state_ = CallbackState::Idle;
+  // Keep track of whether we must send the body when the callback is done
```

Comment:
Is the callback always going to be header processing? Might be worth to note

Reply:
Ack.